src/screens/WebsocketResearchScreen/SampleChatScreen.tsx

```diff
@@ -93,6 +99,11 @@ const SampleChatScreen = () => {
       setLoading(false);
     }
   };
+  React.useEffect(() => {
+    return () => {
+      clearReplyPreview();
+    };
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This `useEffect` hook is used to clear the reply preview when the component unmounts. This is a good practice to clean up any state that might be left over and could potentially cause issues if not cleared.



<!-- This is an auto-generated comment by OSS CodeRabbit -->